src/main/java/App/controller/AuthController.java

```diff
@@ -14,34 +9,33 @@ public class AuthController {
 
     private static AuthenticationService authService= new AuthenticationService();
 
-    public static Long registerNewUser(String name, String email, String password){
+    public static Long registerNewUser(String email, String name, String password){
         if(!checkEmailValid(email)){
-            System.out.println("email field invalid");
+            System.out.println("Invalid Email");
             return null;
         }
         if(!checkNameValid(name)){
-            System.out.println("name field invalid");
+            System.out.println("Invalid Name");
             return null;
         }
         if(!checkPasswordValid(password)){
-            System.out.println("password must include minimum eight characters, at least one letter and one number");
+            System.out.println("Invalid Password. Must include minimum eight characters, " +
+                    "at least one letter and one number");
             return null;
         }
 
-        return authService.createUser(name, email, password);
+        return authService.createUser(email, name, password);
     }
 
     public static Long login(String email, String password){
         if(!checkEmailValid(email)){
             System.out.println("email field invalid");
             return null;
         }
-        if(!checkPasswordValid(password)){
```

Comment:
why remove the message to client about the password?

Reply:
The correctness of the password format is relevant only for registration